go/vt/vtgate/planbuilder/testdata/aggr_cases.txt

```diff
@@ -4171,3 +4171,244 @@ Gen4 error: aggregate functions take a single argument 'count(distinct user_id,
     ]
   }
 }
+
+# having should be able to add new aggregation expressions in having
+"select foo from user group by foo having count(*) = 3"
+"unsupported: filtering on results of aggregates"
+{
+  "QueryType": "SELECT",
+  "Original": "select foo from user group by foo having count(*) = 3",
+  "Instructions": {
+    "OperatorType": "SimpleProjection",
+    "Columns": [
+      0
+    ],
+    "Inputs": [
+      {
+        "OperatorType": "Filter",
+        "Predicate": "[1] = 3",
```

Comment:
it would be nice to have the string representation of that predicate (I.e. `count(*)`)

Reply:
at the filter level, it's reading the column offset, which is information that I think is nice to have. 
we could do something like this:

```
        "Predicate": "offset(1, 'count(*)') = 3",
```

wdyt?

Reply:
I think it is great!